src/main/java/Bob.java

```diff
@@ -0,0 +1,260 @@
+import java.io.IOException;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Scanner;
+
+public class Bob {
+    public static final String LINE_STRING = "____________________________________________________________\n";
+    public static final String DATA_DELIMITER = " @_@ ";
+    public static final String DATA_DIRECTORY = "data";
+    public static final String SAVE_FILE_NAME = "duke.txt";
+    public static final String DIRECTORY_HOME = System.getProperty("user.home");
+    public static TaskList taskList = new TaskList();
+
+
+    public static void main(String[] args) {
+        welcomeMessage();
+        loadFile();
+        scanInput();
+        goodbyeMessage();
```

Comment:
Good use of [SLAP](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard) here.